src/Compilers/CSharp/Test/Emit3/Semantics/PrimaryConstructorTests.cs

```diff
@@ -22650,5 +22650,52 @@ static void validate(ModuleSymbol m)
                     m.GlobalNamespace.GetMember("Test1.<P1>P").GetAttributes().Select(a => a.ToString()));
             }
         }
+
+        [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/70208")]
+        public void AssociatedSymbolOfCapturedPrimaryParameterField()
```

Comment:
>AssociatedSymbolOfCapturedPrimaryParameterField [](http://example.com/codeflow?start=20&length=47)

This test only covers symbol representing original field declaration from source. There are other symbols representing the same entity under different circumstances. Behavior all of them with respect to the affected API(s) should be tested.